README.md

```diff
@@ -80,39 +81,88 @@ pull(
 ## Concepts
 
 Building upon secure-scuttlebutt requires understanding a few concepts
-that it uses to ensure security.
+that it uses to ensure the unforgeability of message feeds.
+
+### Identities
+
+An identity is simply a public/private key pair.
+
+Even though there is no worldwide store of identities, it's infeasible
+for anyone to forge your identity. Identities are binary strings, so not
+particularly human-readable.
+
+### Feeds
+
+A feed is an append-only sequence of messages. Each feed is associated
+1:1 with an identity. The feed is identified by the hash of the public 
+key. This works because public keys are unique.
+
+Since feeds are append-only, replication is simple: request all messages
+in the feed that are newer than the latest message you know about.
 
-### Identity
+Note that append-only really means append-only: you cannot delete an
+existing message. If you want to enable entities to be deleted or 
+modified in your data model, that can be implemented in a layer on top 
+of secure-scuttlebutt using 
+[delta encoding](https://en.wikipedia.org/wiki/Delta_encoding). 
 
-Each node's identity is represented by the hash of their public
-key. Although they are not "human readable", this does
-guarantee that you get unique identifiers (without a central registry)
-and it's infeasible for anyone to forge your identity.
+### Messages
 
-### Secure Data Structures
+Each message contains:
 
-SecureScuttlebutt uses a signed block-chain per identity.
-Each block points to the previous block,
-the signing key, and contains a short message
-and a signature. Every identity has their own block-chain.
+- A message string. This is the thing that the end user cares about.
+- A reference to the previous message. This prevents a malicious party
+  from making a copy of a feed that omits messages in the middle.
```

Comment:
Close but that's not the actual reason for the prevhash. The signature + sequence numbers are enough to stop middlemen from dropping messages. The reason for the prevhash is (and my suggested copy):

```
A content-hash of the previous message. This prevents somebody with the private key from changing
the feed history after publishing, as newly-created message wouldn't match the "prev-hash" of latter messages.
```